README.md

```diff
@@ -81,6 +81,18 @@ wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb && sudo
 `sudo apt-get install esl-erlang`
 + **Install Elixir**: `sudo apt-get install elixir`
 
+#### Windows:
+
++ Web installer
+  + [Download the installer](https://repo.hex.pm/elixir-websetup.exe)
+  + Click next, next, ..., finish
+
++ Chocolatey
```

Comment:
Chocolatey?

Reply:
https://chocolatey.org/ is package manager for Windows. this is commonly used by windows user

Reply:
I think web installer and Chocolatey is same level

Reply:
Would be good to get a clarification in-line fo others. 😉